AK/FloatingPointStringConversions.cpp

```diff
@@ -1669,27 +1625,35 @@ class MinimalBigInt {
 
     void multiply_with_small(u64 value)
     {
-        u64 carry = 0;
-        for (size_t i = 0; i < m_used_length; ++i)
-            m_words[i] = multiply_with_carry(m_words[i], value, carry);
-
-        if (carry != 0)
-            append(carry);
+        if (value <= max_word) {
```

Comment:
Where is this `max_word` (and other things) coming from? Also `StorageOperations`?

Reply:
Oh, I found it.
Could you add an explicit include to this cpp file?

I also don't like the `max_word` name, but that's already merged so that's my problem.